app/services/sign_in/constants/auth.rb

```diff
@@ -15,6 +28,7 @@ module Auth
       GRANT_TYPE = 'authorization_code'
       INFO_COOKIE_NAME = 'vagov_info_token'
       JWT_ENCODE_ALGORITHM = 'RS256'
+      MPI_QUERY_IDENTIFIERS = [ICN = 'ICN', IDME_UUID = 'idme', LOGINGOV_UUID = 'logingov'].freeze
```

Comment:
I don't think we should move over the `MPI_QUERY_IDENTIFIERS`, they're only relevant with the MPI service, so either that service will come with SiS, or we'll keep the existing code that calls it in `vets-api`

Reply:
Got it, I removed the `IDME_UUID`/`LOGINGOV_UUID` ones as well since they're redundant and only used in one place.